server/upgrade_risks_prediction_test.go

```diff
@@ -74,6 +78,73 @@ const upgradeNotRecommended = `
 }
 `
 
+const upgradeRecommendedMultiClusterOk = `
+{
+	"status":"ok",
+	"predictions": [
+		{
+			"cluster_id": "%s",
+			"prediction_status": "ok",
+			"upgrade_recommended": true,
+			"upgrade_risks_predictors": {
+				"alerts": [],
+				"operator_conditions": []
+			},
+			"last_checked_at": "0001-01-01T00:00:00Z"
+		},
+		{
+			"cluster_id": "%s",
+			"prediction_status": "ok",
+			"upgrade_recommended": true,
+			"upgrade_risks_predictors": {
+				"alerts": [],
+				"operator_conditions": []
+			},
+			"last_checked_at": "0001-01-01T00:00:00Z"
+		}
+	]
+}
+`
+
+const upgradeRecommendedMultiClusterTwoOkOneNoData = `
+{
+	"status":"ok",
+	"predictions": [
+		{
+			"cluster_id": "34c3ecc5-624a-49a5-bab8-4fdc5e51a266",
+			"prediction_status": "ok",
+			"upgrade_recommended": true,
+			"upgrade_risks_predictors": {
+				"alerts": [],
+				"operator_conditions": []
+			},
+			"last_checked_at": "0001-01-01T00:00:00Z"
+		},
+		{
+			"cluster_id": "2b9195d4-85d4-428f-944b-4b46f08911f8",
+			"prediction_status": "ok",
+			"upgrade_recommended": true,
+			"upgrade_risks_predictors": {
+				"alerts": [],
+				"operator_conditions": []
+			},
+			"last_checked_at": "0001-01-01T00:00:00Z"
+		},
+		{
+			"cluster_id": "aae0ff10-9892-4572-b77f-73eb3e39825f",
+			"prediction_status": "No data for the cluster",
+			"upgrade_recommended": null,
+			"upgrade_risks_predictors": null,
+			"last_checked_at": null
+		}
+	]
+}
+`
+
+const upgradeMultiClusterTwoClustersNoData = `{"4e550916-1d7b-4434-afec-6125088566f5":{"status":"","data":{"resultType":"","result":null}},"8f7f87e7-cfda-4b55-8517-8d7a089b92f3":{"status":"","data":{"resultType":"","result":null}}}`
```

Comment:
what's this `"data":{"resultType":"","result":null}`? I can't find nothing similar in the code

Reply:
Oh! Good catch!

I used the RHOBS mock output instead of the data eng output. Changing!

Reply:
[fixed, and updated corresponding UT](https://github.com/RedHatInsights/insights-results-smart-proxy/compare/30ce6cb4fb8b57f08cffbe3ca31b4e8efa3d6ada..07bd77001245e2e215d1840c07fe12d0fa9010a0)